fem/pfespace.cpp

```diff
@@ -466,53 +466,54 @@ void ParFiniteElementSpace::ApplyLDofSigns(Table &el_dof) const
    ApplyLDofSigns(all_dofs);
 }
 
-DofTransformation *
-ParFiniteElementSpace::GetElementDofs(int i, Array<int> &dofs) const
+void ParFiniteElementSpace::GetElementDofs(int i, Array<int> &dofs,
```

Comment:
Was there a compelling reason to change the interface here or is it just a dislike of pointers and using NULL to indicate failure?

Reply:
This isn't actually a change of interface since `ParFiniteElementSpace` inherits the declaration with the original signature from `FiniteElementSpace`. This just provides the internal change of functionality.